modules/appeals_api/spec/support/doc_helpers.rb

```diff
@@ -149,8 +150,14 @@ def self.use_shared_schemas?
     !DocHelpers.decision_reviews?
   end
 
+  def self.running_rake_task?
+    # SWAGGER_DRY_RUN is set in the appeals rake tasks: if it's not set, it means the spec is running as part of
+    # a normal rspec suite instead.
+    ENV['SWAGGER_DRY_RUN'].present?
+  end
+
   def self.api_version
-    ENV['RAILS_ENV'] == 'test' ? DEFAULT_CONFIG_VALUES[:api_version] : ENV['API_VERSION']
+    DocHelpers.running_rake_task? ? ENV['API_VERSION'].presence : DEFAULT_CONFIG_VALUES[:api_version]
```

Comment:
This conditional was wrong and broken - the RAILS_ENV seems to be 'test' in all cases where this would be called.